### DISCUSSION TITLE
Smoking

### DISCUSSION THREAD

**Discussion by ellielo** [NAIST-SE/AttractingContributors#342] [General]
Best way to quit smoking in your opinion is ...?

**Comment by Ozankey** [comment]
Smoking is mental problem. Try to understand why you need to quit.

**Comment by hernessteve** [comment]
The best option to don't start smoking. Otherwise - Nicorette gum is good too. You can read more about those gums on [Canada Drugs](https://www.canadadrugsdirect.com/) website if you are interested.  I heard a lot of good things about a famous quit smoking book, never read it though.
